prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -305,7 +305,7 @@ class FakeReport(val metadata: Metadata, val locale : Locale? = null) {
         val rows = (0 until count).map {
             buildRow(schema, roundRobinChoice(states), roundRobinChoice(counties))
         }.toList()
-        return Report(schema, rows, listOf(source))
+        return Report(schema, rows, listOf(source), metadata = metadata)
```

Comment:
The `metadata` file is being used more and more, but is really important for generating a file name, so it's being passed in.